### DISCUSSION TITLE
is there any command to find out whether remote tunneld is running or not?

### DISCUSSION THREAD

**Discussion by mega6453** [doronz88/pymobiledevice3#1056] [Q&A] [answered]
Let say I ran "pymobiledevice3 remote tunneld" command and the window is hidden. So, I just want to know whether tunneld is running or not by running some command in another window. 
 e.g. "pymobiledevice3 remote tunneld isrunning" which will return true or false.  -- This is just an example.. 
 
Thanks!

**Comment by doronz88** [comment]
There currently isn't one - only a REST API
You're welcome to submit a PR for that

**Reply by mega6453** [reply]
Ok, But I couldn't find the REST API in the repo or Readme. Can you help in that? Thanks

**Reply by doronz88** [reply, marked as answer]
https://github.com/doronz88/pymobiledevice3/blob/8e96de30a373ffc003e4babf0f94d39e62bd6a73/pymobiledevice3/tunneld.py#L356

**Reply by mega6453** [reply]
Yes it works. Thanks. 
But one more question. 

http://127.0.0.1:49151/hello - will this port be same always or will it change in case if that port is busy?
